### PR TITLE
Setup project

### LINKED ISSUE #7: Setup Clean Architecture

### PR DESCRIPTION
**Pull request by EntyR**
setup clean architecture modules change gradle config and package name

### CODE REVIEW

File: data/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (AlexeyKolosov) [inline data/.gitignore]: why did you add gitignore to data folder ?

File: domain/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (AlexeyKolosov) [inline domain/.gitignore]: the same here, lets have only one gitignore

File: data/.gitignore (hunk 1 above)
Comment (EntyR) [reply data/.gitignore]: was autogenerated by android sudio, now it's deleted

File: domain/.gitignore (hunk 2 above)
Comment (EntyR) [reply domain/.gitignore]: done

**Review by AlexeyKolosov — APPROVED** [review]
